### ISSUE TITLE
BUG: Mermaid Timeline graph in Excalidraw plugin has all black

### ISSUE THREAD

**Issue by madukan** [zsviczian/obsidian-excalidraw-plugin#1424]
**Your environment**
Obsidian `Command Palette/Show Debug info`:

```
SYSTEM INFO:
	Obsidian version: v1.4.16
	Installer version: v1.4.13
	Operating system: Darwin Kernel Version 23.0.0: Fri Sep 15 14:41:43 PDT 2023; root:xnu-10002.1.13~1/RELEASE_ARM64_T6000 23.0.0
	Login status: not logged in
	Insider build toggle: off
	Live preview: on
	Legacy editor: off
	Base theme: dark
	Community theme: Things v2.1.18
	Snippets enabled: 0
	Restricted mode: off
	Plugins installed: 4
	Plugins enabled: 4
		1: Excalidraw v1.9.28
		2: Dataview v0.5.64
		3: Kanban v1.5.3
		4: Obsidian Git v2.22.0

RECOMMENDATIONS:
	Custom theme and snippets: for cosmetic issues, please first try updating your theme and disabling your snippets. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community theme and snippets.
	Community plugins: for bugs, please first try updating all your plugins to latest. If still not fixed, please try to make the issue happen in the Sandbox Vault or disable community plugins.
```

**Describe the bug**
When creating a Timeline (https://mermaid.js.org/syntax/timeline.html) in Dark Mode of Obsidian (and macOS), the graphs seem all black and there is no way to go back to the diagram to update anything as well. Perhaps they are two different issues but sounds like a bug.

**To Reproduce**
1. Open or create an Excalidraw diagram in Obsidian
2. In the tool box of Excalidraw plugin, select "More tools"
3. Select Mermaid to Excalidraw option
4. Set a timeline value. Eg:
```
timeline
    title History of Social Media Platform
    2002 : LinkedIn
    2004 : Facebook
         : Google
    2005 : Youtube
    2006 : Twitter
```
Note: 
a. You may get as many variations from https://mermaid.js.org/syntax/timeline.html
b. Above diagram will be interpreted by Github so you may not see the code. Use the copy button in the diagram to get the code.

5. Note that the diagram is all blacked out in the boxes and can't read the text.

**Expected behavior**
1. Colours as defined in the MermaidJS library (or in Obsidian)
in Obsidian note (markdown file) you can try this, to see how that respond:
<img width="402" alt="image" src="https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/7971561/812441a3-f0e9-44a2-8b69-55703802b166">

Usual interpretation (now this one is from Github indeed)
```mermaid
timeline
    title History of Social Media Platform
    2002 : LinkedIn
    2004 : Facebook
         : Google
    2005 : Youtube
    2006 : Twitter
```
Notice the colours and contrast, you should be able to see accurately.

2. Ability to edit the Mermaid value and update the graph in Excalidraw plugin.

**Screenshots**

Issue view in Excalidraw plugin:
<img width="660" alt="image" src="https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/7971561/71493673-605c-472b-a39c-abb490e92404">

Expected interpretation in Obsidian page:

<img width="730" alt="image" src="https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/7971561/5951eeb0-434c-45bd-8a53-ac5395ff4318">

Indeed Github interpretation too looks good.

**Comment by zsviczian** [comment]
fixed in 2.0.0 - will be released this weekend

**Comment by madukan** [comment]
Thank you 🙏
